Clean CaDET Platform.sln

```diff
@@ -17,7 +17,9 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "SmartTutorTests", "SmartTut
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "DataSetExplorer", "DataSetExplorer\DataSetExplorer.csproj", "{5064A050-8C9B-4AAC-8F5C-09BEDCFCF4C8}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "DataSetExplorerTests", "DataSetExplorerTests\DataSetExplorerTests.csproj", "{E6AA4545-2AB6-42E0-B560-612F64667366}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "DataSetExplorerTests", "DataSetExplorerTests\DataSetExplorerTests.csproj", "{E6AA4545-2AB6-42E0-B560-612F64667366}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "SourceCodeTests", "SourceCodeTests\SourceCodeTests.csproj", "{3463662F-E9C5-4F5C-9981-A84775C10D14}"
```

Comment:
What is the reasoning behind a completely new project? Since this functionality is tied to the challenges, can we not include it in the SmartTutorTests?

Reply:
This was a misunderstanding in communication. This will be moved.